core/src/main/resources/filter.properties

```diff
@@ -0,0 +1,3 @@
+kw1=TransactionImple
+kw2=Periodic Recovery
+kw3=TransactionSynchronizationRegistryImple
```

Comment:
can it be such like "keywords=TransactionImple, Periodic Recovery,TransactionSynchronizationRegistryImple" ?
and you can use String.split(",") to get the words